.github/workflows/on_run_publish_surefire_report.yml

```diff
@@ -50,6 +55,14 @@ jobs:
           with:
             path: ./github-sha.txt
 
+        - name: Remove flaky tests from failure report
```

Comment:
Aren't the flaky tests always reported in files which contain `FLAKY` in the filename, instead of installing `xmlstarlet`, can't we use a glob similar to the Jenkinsfile?

Reply:
there's a maven plugin that generates the FLAKY files from the surefire report, but flaky tests are not removed form the original report they are just copied, that is what confused me.
they need to be removed manually otherwise they are counted are failures